modules/pytket-ionq/tests/backend_test.py

```diff
@@ -85,8 +81,9 @@ def test_invalid_request() -> None:
         assert "does not satisfy GateSetPredicate" in str(excinfo.value)
 
 
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
```

Comment:
Again, these tests shouldn't be skipped. Overall, there is only one test running in CI now (see for instance [here](https://github.com/CQCL/pytket-extensions/runs/6031819939?check_suite_focus=true))